pom.xml

```diff
@@ -1470,5 +1470,31 @@
       </build>
     </profile>
 
+    <profile>
+      <id>pitest-size</id>
```

Comment:
@Efremenkov for the out of bounds exception, did you try the workaround listed in https://github.com/checkstyle/checkstyle/issues/3271#issuecomment-255500681 ?
That fixed the problem for me.

Grammar might be picking up our antlr grammar tests, which might explain the extra time. Those files are generated and shouldn't be in pitest.
The only true class in grammar package is `CommentListener` and it is an interface, so really there is nothing to test in grammar package.